keras_cv/models/__internal__/unet.py

```diff
@@ -177,6 +177,12 @@ def UNet(
     All function parameters require curried functions as inputs which return a
     function that acts on tensors as inputs.
 
+    Reference:
+        - [U-Net: Convolutional Networks for Biomedical Image Segmentation](https://arxiv.org/abs/1505.04597)
+
+            U-Net relies on strong use of Data Augmentation and architecture consists of a contracting path to capture 
```

Comment:
I think we can drop this second sentence, but the reference part LGTM -- thanks!

Reply:
Sorry I think I wasn't clear -- let's only include the link (and drop the part that starts with "U-Net relies on strong use ..."